data/data-pipeline/data_pipeline/score/field_names.py

```diff
@@ -196,3 +197,58 @@
 HOLC_GRADE_D_TRACT_20_PERCENT_FIELD: str = "Tract is >20% HOLC Grade D"
 HOLC_GRADE_D_TRACT_50_PERCENT_FIELD: str = "Tract is >50% HOLC Grade D"
 HOLC_GRADE_D_TRACT_75_PERCENT_FIELD: str = "Tract is >75% HOLC Grade D"
+
+
+# Climate Change
+FEMA_LOSS_RATE_LOW_INCOME = f"At or above the {PERCENTILE}th percentile for FEMA's expected loss rate and is low income"
+AGRICULTURE_LOSS_RATE_LOW_INCOME = f"At or above the {PERCENTILE}th percentile for agriculture and is low income"
+EXPECTED_BUILDING_LOSS_RATE_LOW_INCOME = f"At or above the {PERCENTILE}th percentile for building loss and is low income"
+
+# Clean energy and efficiency
```

Comment:
Same comments as above: can we make these consistent? They're all over the map right now.

Reply:
Understood, yes